### DISCUSSION TITLE
--subs-srt, --subtitles-srt

### DISCUSSION THREAD

**Discussion by iltifat86** [FOSWLY/vot-cli#34] [General]
im use vot-cli version 1.4.1 and args "-subs-srt" not working
help me fix it pls

**Comment by ilyhalight** [comment]
Update to vot-cli 1.4.3

**Reply by iltifat86** [reply]
> Update to vot-cli 1.4.3

это тролинг такой?) последняя вроде 1.4.1

**Reply by ilyhalight** [reply]
> > Update to vot-cli 1.4.3
> 
> последняя вроде 1.4.1

На гитхабе забыл добавить в релизы, в нпм уже давно есть
